sdk/cosmos/azure-cosmos/azure/cosmos/_session.py

```diff
@@ -205,9 +195,11 @@ def parse_session_token(response_headers):
 
 
 class Session(object):
-    """
-    State of a Azure Cosmos session. This session object
-    can be shared across clients within the same process
+    """State of a Azure Cosmos session.
+
+    This session object can be shared across clients within the same process
```

Comment:
Final period.

Reply:
Actually periods missing throughout this module.

Reply:
fixed in https://github.com/Azure/azure-sdk-for-python/pull/8607/commits/c14bb0d8adcb5e63ac1d6a2fab3819ccca50707e